extensions/binding/org.eclipse.smarthome.binding.hue.test/src/test/java/org/eclipse/smarthome/binding/hue/internal/LightStateConverterTest.java

```diff
@@ -101,6 +101,17 @@ public void hsbBrightnessAlwaysGreaterThanZero() {
         }
     }
 
+    @Test
+    public void hsbHueAlwaysGreaterThanZeroAndLessThan360() {
+        final State lightState = new State();
+        lightState.colormode = ColorMode.CT.toString();
```

Comment:
We maybe want to use `ColorMode.HS.toString()` in this case too - which is default and you do not need to set it.

Reply:
changed, thx.